libs/designsystem/src/lib/components/calendar/calendar.component.ts

```diff
@@ -405,4 +445,25 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
       month: this.month,
     };
   }
+
+  // TODO: Should this be based on new Date() (so deterministically today) or this.todayDate?
```

Comment:
Both - it depends on whether `todayDate` is set or not. So implementation is fine 👍 
```suggestion
```